src-electron/generator/helper-session.js

```diff
@@ -612,9 +613,12 @@ function all_user_cluster_commands_irrespective_of_manufaturing_specification(
 
 /**
  * Creates endpoint type cluster attribute iterator. This fetches all
- * manufacturing and non-manufaturing specific attributes which have been enabled
- * on added endpoints
+ * manufacturing and non-manufaturing specific attributes which have been
```

Comment:
What is "non-manufacturing" (which is not spelled right here)?

Reply:
The standard cluster attributes

Reply:
I think that should be rephrased to be clear, then.